_topics/en/anchor-outputs.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Anchor outputs
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - Simplified commitments
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightning Network
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Anchor outputs** are special outputs in LN commitment transactions
+  that are designed to allow the transaction to be fee bumped.  An
+  earlier name for the proposal was **simplified commitments.**
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Each time the balance changes in an LN channel, a *commitment
+  transaction* is created and signed by the participating parties.  The
+  transaction is only broadcast if one party decides they want to
+  unilaterally close the channel (e.g. because the other party has
+  become unresponsive).  Because the broadcast of the commitment
+  transaction may occur a long time after it was created, the commitment
+  transaction may pay too much or too little in transaction fees.
+  Paying a too-low feerate may prevent the commitment transaction from
+  confirming before any timelocks contained within it expire, allowing
+  funds to be stolen.
+
+  The solution to this is for the commitment transaction to pay a
+  minimal amount of fees and then to allow the participating user to fee
```

Comment:
Heh, I managed to confuse both of you, so I'll definitely change this.  "Participating user" was meant to refer to either user participating in the channel.

- participating user<b>s</b> doesn't work because that may imply both users need to cooperate to fee bump, which doesn't work for a unilateral close
- **channel-closing user** doesn't work because the non-closing user also needs to be able to fee bump or there's the problem of the closing user setting a really low fee in order to get any HTLCs to expire

Reply:
I like the latest change to "either channel participant"